packages/babel-plugin/src/constants.tsx

```diff
@@ -1 +1,3 @@
 export const PROPS_IDENTIFIER_NAME = 'props';
+export const COMPILED_DIRECTIVE_DISABLE_SAME_LINE = '@compiled-disable';
```

Comment:
`@compiled-disable` > `@compiled-disable-line` makes this in-line with eslint

Then rename `COMPILED_DIRECTIVE_DISABLE_SAME_LINE` to `COMPILED_DIRECTIVE_DISABLE_LINE`